mirage/seed_image/fits_seed_image.py

```diff
@@ -375,6 +377,7 @@ def save_seed_image(self, model):
         dim = len(self.seed_image.shape)
         if dim == 2:
             units = 'e-/sec'
+            # TODO(eslavich): Where should this come from?
             yd,xd = arrayshape
```

Comment:
This is the line that I don't know for sure how to fix.  Should I just replace `arrayshape` with `dim`?

Reply:
I wouldn't worry about fits_seed_image.py, nor blot_image.py. I'm in the middle of basically completely rewriting those. I put them together for a particular case for someone a long while ago and haven't touched them since, so they are in serious need of updating.

Reply:
Okey dokey, I'll just remove my comment